features/git-new-pull-request/github.feature

```diff
@@ -5,7 +5,7 @@ Feature: git-new-pull-request when origin is on GitHub
   So that I have more time for coding the next feature instead of wasting it with process boilerplate.
 
 
-  Scenario Outline: result
+  Scenario Outline: creating pull-requests
```

Comment:
This seems unrelated and the other feature files should probably be updated in a similar way. Would be good to do in a separate PR.